generate-stackbrew-library.sh

```diff
@@ -2,15 +2,15 @@
 set -Eeuo pipefail
 
 declare -A aliases=(
-	[2.4]='2 latest'
-	[2.5-rc]='rc'
+	[2.5]='2 latest'
```

Comment:
Probably we want `[2.5]='2 latest 2.5-rc rc'` here (i. e. `2.5-rc` & `rc` images should be updated to `2.5`).